daemon/cmd/daemon_main.go

```diff
@@ -348,6 +348,15 @@ func InitGlobalFlags(cmd *cobra.Command, vp *viper.Viper) {
 	flags.Bool(option.EnableL7Proxy, defaults.EnableL7Proxy, "Enable L7 proxy for L7 policy enforcement")
 	option.BindEnv(vp, option.EnableL7Proxy)
 
+	flags.Bool(option.EnableDropNotify, defaults.EnableDropNotify, "Enable by default DropNotification on any endpoint")
```

Comment:
The phrase "by default" here and elsewhere in this PR isn't accurate. It is implied that a flag has a default value, and the value of this flag controls the runtime behavior for all endpoints, it doesn't control the default.

That is, the description of this flag, and elsewhere in the PR should should be along the lines of "Enable DropNotification for any endpoint".